pkg/frontend/adminactions/kubeactions.go

```diff
@@ -69,9 +78,15 @@ func NewKubeActions(log *logrus.Entry, env env.Interface, oc *api.OpenShiftClust
 
 		dyn:       dyn,
 		configcli: configcli,
+		kubecli:   kubecli,
 	}, nil
 }
 
+func (k *kubeActions) KubeGetPodLogs(ctx context.Context, namespace, podName, containerName string) ([]byte, error) {
+	opts := corev1.PodLogOptions{Container: containerName}
```

Comment:
Ohh yes .. that is a very good point .. I will add that limit ..

Reply:
Noting your point about trimming the slice ..I am now rather than trimming the logs, I am just adding a new line character to the result .. hence eliminating any chance of `panic slice bounds out of range`, so doing the same thing in a different and more safer way.
I will push the changes in some time ..